Oral Metagenomics
----------
Periodontitis has been recently defined as a dysbiotic disease resulting from imbalanced oral microbiota. The transition of microbial communities from commensal to periodontitis-associated ones likely requires colonization by specific pathogens, including Porphyromonas gingivalis. Given the importance of P. gingivalis as a keystone pathogen of polymicrobial communities, the determinants of P. gingivalis levels, its interaction with the core microbiota, and association with the pathogenic potential of the microbial communities need to be addressed.

This present study intends to determine the role of Streptococcus cristatus in altering interactions of P. gingivalis with other oral bacteria in a complex context. We collected dental plaque samples from periodontitis patients and assigned them into two groups based on their ratios of S. cristatus and P. gingivalis. We then characterized microbial profiles of the dental plaque samples using shotgun metagenomic sequencing and subsequently compared oral microbial composition and functional capabilities between groups with high or low S. cristatus-P. gingivalis ratios. Our metagenomic sequencing data are deposited at Sequence Read Archive (SRA) with accession number PRJNA1010809 (https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1010809).  

The Python code provided here was used to generate some of the results for a preprint paper entitled "Oral microbiome associated with differential ratios of Porphyromonas gingivalis and Streptococcus cristatus".

Citation
----------
Q. Wang, BY. Wang, S. Pratap, H. Xie. Oral microbiome associated with differential ratios of Porphyromonas gingivalis and Streptococcus cristatus. Res. Sq. [Preprint], https://doi.org/10.21203/rs.3.rs-3266326/v1, 2023.
